src/app/clusters/icd-management-server/icd-management-server.cpp

```diff
@@ -59,21 +65,50 @@ CHIP_ERROR IcdManagementAttributeAccess::Read(const ConcreteReadAttributePath &
 
     switch (aPath.mAttributeId)
     {
+    case IcdManagement::Attributes::IdleModeInterval::Id:
```

Comment:
Please add these attributes to the attributeAccessInterfaceAttributes sections in `src/app/zap-templates/zcl/zcl.json` and `src/app/zap-templates/zcl/zcl-with-test-extensions.json`

Reply:
Added.